extensions/exposition/documentation/vary.md

```diff
@@ -0,0 +1,39 @@
+# HTTP request details
+
+The vary directives family provides the capability to include HTTP request details as input for an
```

Comment:
```suggestion
The `vary` directives family provides the capability to include HTTP request headers as input for an
```

Reply:
That would be incorrect. As I've [mentioned](https://github.com/toa-io/toa/issues/480#issuecomment-1920730698) earlier, headers are not the sole request details available through this mechanism..